apps/fz_http/lib/fz_http/telemetry.ex

```diff
@@ -104,14 +114,22 @@ defmodule FzHttp.Telemetry do
     |> Keyword.get(:host)
   end
 
-  defp distinct_id do
-    Application.fetch_env!(:fz_http, :telemetry_id)
-  end
-
   defp version do
     Application.spec(:fz_http, :vsn) |> to_string()
   end
 
+  defp external_database? do
+    db_host = conf(FzHttp.Repo)[:hostname]
+
+    db_host != "localhost" && db_host != "127.0.0.1"
+  end
```

Comment:
Since this is a non-user-facing part of the codebase, I think we may want to handle more edge cases such as when a [`url` is specified instead of the `hostname`](https://github.com/firezone/firezone/blob/master/config/dev.exs#L6). It's currently only in `dev` but can see myself forgetting to update this if we ever move that var to prod (e.g. for Docker deployments).

```suggestion
  defp external_database?(repo_conf) when is_map_key(repo_conf, :url) do
    %{host: host} = URI.parse(repo_conf.url)
    is_local_db?(host)
  end
  
  defp external_database?(repo_conf) when is_map_key(repo_conf, :hostname) do
    is_local_db?(repo_conf.hostname)
  end
  
  defp is_local_db?(host) do
    host == "localhost" || host == "127.0.0.1"
  end
  
  def ping_data do
    # ...
    external_database?(conf(FzHttp.Repo))
    # ...
  end
```

Reply:
Cool, I can do that 👍 

We'd want to invert the `is_local_db?` method to match the ping data field name `external_database`.  Also I'm fine just changing the field to `local_database` if that's more clear.

Reply:
Sure